OData/src/System.Web.OData/OData/EnableQueryAttribute.cs

```diff
@@ -590,7 +590,7 @@ private object ExecuteQuery(object response, HttpRequestMessage request, HttpAct
 
             // apply the query
             IEnumerable enumerable = response as IEnumerable;
-            if (enumerable == null)
+            if (enumerable == null || (response is string) == true || (response is byte[]) == true)
```

Comment:
For people who are moderately vision impaired (like myself, for example), using the full Boolean semantics is much easier to read and avoids many issues with readability,